### PR TITLE
RaisimGymTorch: Make picklable, expose reward information to Python

### PR DESCRIPTION
**Pull request by wxmerkt**
Hi @jhwangbo 
We are using RaisimGymTorch as a drop-in to OpenAI Gym and made a few adjustments:

1. The environment is now picklable, where the state is described by the resource directory and the config file.
2. We also store the reward-reward as a map in addition to the reward-integral, and we expose them to Python
3. The kwargs of the constructor are now named.
4. We explicitly call `init()` in the constructor - this way the separate call to `init()` from Python can be omitted.

Looking forward to your feedback,
Wolfgang

### CODE REVIEW

**Comment by jhwangbo** [comment]
Thank you for your contribution. The modification is great. I had a plan to do this as well. I made a small comment on the naming convention. After you resolve it, we can merge it to the master

**Comment by wxmerkt** [comment]
Thanks, I can't see the comment - could you please add it again and I am happy to address

File: raisimGymTorch/raisimGymTorch/env/Reward.hpp [hunk 1]
```diff
@@ -76,9 +76,18 @@ class Reward {
...
+      reward_map_[rw.first] = rw.second.reward;
+    reward_map_["reward_sum"] = sum();
+
+    return reward_map_;
+  }
+
  private:
   std::map<std::string, raisim::RewardElement> rewards_;
   std::map<std::string, float> costSum_;
+  std::map<std::string, float> reward_map_;
```
[the remark this answers is not in the document]
Comment (jhwangbo) [reply raisimGymTorch/raisimGymTorch/env/Reward.hpp:90]: I forgot to press submit review. Now you should see this ;)

[the remark this answers is not in the document]
Comment (wxmerkt) [reply raisimGymTorch/raisimGymTorch/env/Reward.hpp]: Thank you, I just amended

**Comment by jhwangbo** [comment]
I meant for all variables. I still see many snake_case in C++.

**Comment by jhwangbo** [comment]
and for all methods as well

File: raisimGymTorch/raisimGymTorch/env/Reward.hpp [hunk 2]
```diff
@@ -76,11 +76,20 @@ class Reward {
     return costSum_;
   }
 
+  const std::map<std::string, float>& getStdMapOfRewardReward() {
```
Comment (jhwangbo) [inline raisimGymTorch/raisimGymTorch/env/Reward.hpp:79]: not sure if this is the best name. Maybe just ``getStdMap()`` is better

**Comment by wxmerkt** [comment]
Sorry for missing this earlier - I just renamed all introduced variables and methods to camelCase

**Comment by jhwangbo** [comment]
thank you!

**Pull request merged**